NEWS.md

```diff
@@ -1,3 +1,7 @@
+# mutscan 0.2.27
+
+* Add digestBarcodes function
```

Comment:
Just a thought about the name of the `digestBarcodes` function: For me it is relatively close to `digestFastqs` function name, and one could be confused which one to use when. For example, there may be cases where the fastq file may only contain barcodes.

Do you think it could be worth looking for a name that is easier to distinguish from `digestFastqs`, and maybe more specifically describes the use case for this function? Maybe something along the lines `linkBarcodes`, `linkVariants`, `linkMultipleVariants`, `digestIndividualVariants` or similar?

Reply:
Yes, totally agree that this is unclear. I like the names with `link` - it would also indicate the different type of output compared to `digestFastqs`.

Reply:
Sounds good, what about `linkMultipleVariants` then? I am happy to do the renaming while I am at it.

Reply:
Perfect, thanks!

Reply:
resolved in 7d10aba